sources/ReplicantWindow.cpp

```diff
@@ -7,16 +7,20 @@
  */
 
 
+#include <Catalog.h>
 #include <ControlLook.h>
 #include <LayoutBuilder.h>
 
 #include "DropZoneTab.h"
 #include "ReplicantWindow.h"
 
+#undef B_TRANSLATION_CONTEXT
+#define B_TRANSLATION_CONTEXT "ReplicantWindow"
+
 
 ReplicantWindow::ReplicantWindow(BRect frame)
 	:
-	BWindow(BRect(0, 0, 120, 100), ("Resize"),
+	BWindow(BRect(0, 0, 120, 100), (B_TRANSLATE("Resize")),
```

Comment:
But it would still truncate with font size 24. I guess you want the title so that the window can be closed easily. I can't come up with a better idea. Maybe take care of this in a future commit?